### PR TITLE
Add init-sdmmc

### PR DESCRIPTION
**Pull request by DanielHo-BS**

### CODE REVIEW

File: sdmmc/copy-rootfs-sdmmc.sh [hunk 1]
```diff
@@ -1,6 +1,4 @@
-#!/bin/bash
-# Mount the SDMMC as /mnt
-sudo mkfs.ext4 /dev/mmcblk1p1
+sudo mkdir /mnt
```
Comment (evshary) [inline sdmmc/copy-rootfs-sdmmc.sh]: I think we don't need to recreate /mnt folder.

File: sdmmc/init-sdmmc.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# Mount the SDMMC as /mnt
+sudo umount /dev/mmcblk1p1
```
Comment (evshary) [inline sdmmc/init-sdmmc.sh]: We don't need to split the script into two parts.

**Review by evshary — CHANGES_REQUESTED** [review]

File: sdmmc/copy-rootfs-sdmmc.sh (hunk 1 above)
Comment (DanielHo-BS) [reply sdmmc/copy-rootfs-sdmmc.sh]: OK.

File: sdmmc/init-sdmmc.sh (hunk 2 above)
Comment (DanielHo-BS) [reply sdmmc/init-sdmmc.sh]: OK

**Review by DanielHo-BS — COMMENTED** [review]
Done the request changes

**Comment by evshary** [comment]
LGTM

**Pull request merged**